src/sbd-common.c

```diff
@@ -118,6 +118,64 @@ usage(void)
                 , cmdname);
 }
 
+#define MAX_WATCHDOGS 64
+#define SYS_CLASS_WATCHDOG "/sys/class/watchdog"
+#define SYS_CHAR_DEV_DIR "/sys/dev/char"
+#define WATCHDOG_NODEDIR "/dev/"
+
+#define WATCHDOG_NODEDIR_LEN 5
```

Comment:
FYI sizeof(WATCHDOG_NODEDIR) would work here and be optimized out by compiler

Reply:
sizeof(WATCHDOG_NODEDIR) - 1 more precisely ;-)
In for a long time so I don't remember if there was a reason for going that route.